Source/WTF/wtf/URLParser.h

```diff
@@ -124,8 +124,7 @@ class URLParser {
     bool hasForbiddenHostCodePoint(const LCharBuffer&);
     void percentEncodeByte(uint8_t);
     void appendToASCIIBuffer(char32_t);
-    void appendToASCIIBuffer(const char*, size_t);
-    void appendToASCIIBuffer(const LChar* characters, size_t size) { appendToASCIIBuffer(reinterpret_cast<const char*>(characters), size); }
+    void appendToASCIIBuffer(std::span<const LChar>);
```

Comment:
Do we not need the dedicated `char*` overload? If not, why do we ever declare `LChar` arguments and `char` arguments?

Reply:
We don't need it because it is not getting called anymore. `LChar` and `char` are still different. `LChar` is currently a `uint8_t`.